test/circle/lint.sh

```diff
@@ -30,6 +30,67 @@ run() {
   fi
 }
 
+# Lint the Chart.yaml and values.yaml files for Helm
+yamllinter() {
+  printf "\nLinting the Chart.yaml and values.yaml files at ${1}\n"
+
+  # If a Chart.yaml file is present lint it. Otherwise report an error
+  # because one should exist
+  if [ -e $1/Chart.yaml ]; then
+    run yamllint -c test/circle/lintconf.yml $1/Chart.yaml
+  else
+    echo "Error $1/Chart.yaml file is missing"
+    exitCode=1
+  fi
+
+  # If a values.yaml file is present lint it. Otherwise report an error
+  # because one should exist
+  if [ -e $1/values.yaml ]; then
+    run yamllint -c test/circle/lintconf.yml $1/values.yaml
+  else
+    echo "Error $1/values.yaml file is missing"
+    exitCode=1
+  fi
+}
+
+# Verify that the semver for the chart was increased
+semvercompare() {
+  printf "\nChecking the Chart version has increased for the chart at ${1}\n"
+
+  # Checkout the Chart.yaml file on master to read the version for comparison
+  git show k8s/master:$1/Chart.yaml > /tmp/Chart.yaml
+
+  local oldVer=`yaml r /tmp/Chart.yaml version`
+  local newVer=`yaml r $1/Chart.yaml version`
+
+  # Pre-releases may not be API compatible. So, when tools compare versions
```

Comment:
We should probably review what charts are using pre-releases and move them away from it. There shouldn't be any charts in stable using pre-release versions and those are using it to denote the version of the packaged app should adopt appVersion instead.